fusor-ember-cli/app/components/base-f.js

```diff
@@ -16,11 +16,11 @@ export default Ember.Component.extend({
   },
 
   labelClassSize: Ember.computed(function () {
-    return this.getWithDefault('labelSize', 'col-lg-2 col-md-3 col-sm-5');
+    return this.getWithDefault('labelSize', 'col-lg-3 col-md-3 col-sm-4 col-xs-4 col-xs-offset-1');
```

Comment:
@cfchase, at first I just modified the naming-scheme-modal only, but then I I changed to modify the default. Otherwise, we could just remove the default if they are always overwritten. I can move col-sm back to -7 and -5. 
The fix on the PR was adding `xs` sizes and the `xs-offset-1` for the label.